libs/product/package.json

```diff
@@ -27,6 +27,7 @@
     "@angular/common": "0.0.0-PLACEHOLDER",
     "@angular/core": "0.0.0-PLACEHOLDER",
     "@daffodil/core": "0.0.0-PLACEHOLDER",
+    "@daffodil/product": "0.0.0-PLACEHOLDER",
```

Comment:
was this a mistake?

Reply:
no that needs to be there. I can move it to a different PR though

Reply:
the product package is dependent on itself?

Reply:
oh.... haha yeah it was a mistake then